packages/ui/src/components/organisms/Form/_Validator/value-validators/document.value.validator.ts

```
@@ -0,0 +1,42 @@
+export class DocumentValueValidator extends ValueValidator<IDocumentValueValidatorParams> {
+  type = 'document';
+
+  validate(_: unknown, fieldContext: IFieldContext): void {
+    const { pathToDocuments, documentId, pageIndex = 0 } = this.params;
+
+    const documentsPath = this.getDocumentsPathWithIndexes(pathToDocuments, fieldContext);
+    const documents = get(fieldContext.context, documentsPath);
+
+    const document = documents?.find((document: Document) => document.id === documentId);
+
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)